src/common/aes/aes.h

```diff
@@ -147,6 +149,32 @@ void OQS_AES256_CTR_inc_stream_iv(const uint8_t *iv, size_t iv_len, const void *
  */
 void OQS_AES256_CTR_inc_stream_blks(void *ctx, uint8_t *out, size_t out_blks);
 
+struct OQS_AES_callbacks {
+	void (*AES128_ECB_load_schedule)(const uint8_t *key, void **ctx);
+	void (*AES128_free_schedule)(void *ctx);
+	void (*AES128_ECB_enc)(const uint8_t *plaintext, const size_t plaintext_len, const uint8_t *key, uint8_t *ciphertext);
+	void (*AES128_ECB_enc_sch)(const uint8_t *plaintext, const size_t plaintext_len, const void *schedule, uint8_t *ciphertext);
+	void (*AES256_ECB_load_schedule)(const uint8_t *key, void **ctx);
+	void (*AES256_CTR_inc_init)(const uint8_t *key, void **ctx);
+	void (*AES256_CTR_inc_iv)(const uint8_t *iv, size_t iv_len, void *ctx);
+	void (*AES256_CTR_inc_ivu64)(uint64_t iv, void *ctx);
+	void (*AES256_free_schedule)(void *ctx);
+	void (*AES256_ECB_enc)(const uint8_t *plaintext, const size_t plaintext_len, const uint8_t *key, uint8_t *ciphertext);
+	void (*AES256_ECB_enc_sch)(const uint8_t *plaintext, const size_t plaintext_len, const void *schedule, uint8_t *ciphertext);
+	void (*AES256_CTR_inc_stream_iv)(const uint8_t *iv, size_t iv_len, const void *ctx, uint8_t *out, size_t out_len);
+	void (*AES256_CTR_inc_stream_blks)(void *ctx, uint8_t *out, size_t out_blks);
+};
+
+/**
+ * Set callback functions for AES operations.
+ *
+ * This function may be called at most once before OQS_init to replace
```

Comment:
Why "at most" (once callable)? What if it's never called? What if it's called more than once?

Reply:
Yes, it can be called multiple times and if it's not called, the default implementation is used. I've updated the doc comment.